src/TLuaInterpreter.cpp

```diff
@@ -11955,6 +11955,55 @@ int TLuaInterpreter::sendIrc( lua_State * L )
     return 0;
 }
 
+int TLuaInterpreter::setServerEncoding(lua_State * L)
```

Comment:
May be an issue when we allow the GUI to be language dependent - wanted to keep the two things clearly separate.  Also, at least keyboards could have encodings and the term `encoding` might be used in other contexts e.g. cryptography by tying it to server I hope to avoid later confusions.